plugins/meta/tuning/tuning.go

```diff
@@ -54,10 +60,15 @@ type TuningConf struct {
 }
 
 type IPAMArgs struct {
-	SysCtl  *map[string]string `json:"sysctl"`
-	Mac     *string            `json:"mac,omitempty"`
-	Promisc *bool              `json:"promisc,omitempty"`
-	Mtu     *int               `json:"mtu,omitempty"`
+	SysCtl   *map[string]string `json:"sysctl"`
+	IfConfig *ifConfig          `json:"ifConfig,omitempty"`
```

Comment:
This would be changing the configuration format, which we shouldn't do. Do you just want to embed the IfConfig struct?

Reply:
This was changed because of @bboreham request: https://github.com/containernetworking/plugins/pull/540#discussion_r513554021
I have explained this under that comment. If you need any more explanation, don't hesitate to ask.

Should I revert this change?

Reply:
The explanation helps, but it does not override the point that we can't just make a breaking change to the API because you think it's nicer.
I did say "maybe"; if it can't be done, so be it.

Reply:
@squeed @bboreham Just reverted the API changes.